howto/unicode.po

```diff
@@ -219,35 +231,37 @@ msgid ""
 "Therefore this encoding isn't used very much, and people instead choose "
 "other encodings that are more efficient and convenient, such as UTF-8."
 msgstr ""
+"Par conséquent, cet encodage n'est pas très utilisé, et les gens choisissent "
+"plutôt d'autres encodages qui sont plus efficaces et pratiques, comme UTF-8."
 
 #: ../Doc/howto/unicode.rst:125
-#, fuzzy
 msgid ""
 "UTF-8 is one of the most commonly used encodings, and Python often defaults "
 "to using it.  UTF stands for \"Unicode Transformation Format\", and the '8' "
 "means that 8-bit values are used in the encoding.  (There are also UTF-16 "
 "and UTF-32 encodings, but they are less frequently used than UTF-8.)  UTF-8 "
 "uses the following rules:"
 msgstr ""
-"UTF-8 est l’un des encodages les plus couramment utilisés. UTF signifie "
-"« Unicode Transformation Format » (format de transformation Unicode) et "
-"« 8 » signifie que des nombres à 8 bits sont utilisés dans l'encodage. (Il "
-"existe également des codages UTF-16 et UTF-32, mais ils sont moins souvent "
-"utilisés que UTF-8.) UTF-8 utilise les règles suivantes :"
+"UTF-8 est l’un des encodages les plus couramment utilisés, et Python "
+"l’utilise souvent par défaut. UTF signifie « Unicode Transformation "
+"Format » (format de transformation Unicode) et « 8 » signifie que des "
+"nombres à 8 bits sont utilisés dans l'encodage. (Il existe également des "
+"codages UTF-16 et UTF-32, mais ils sont moins souvent utilisés que UTF-8.) "
+"UTF-8 utilise les règles suivantes :"
 
 #: ../Doc/howto/unicode.rst:131
 msgid ""
 "If the code point is < 128, it's represented by the corresponding byte value."
 msgstr ""
-"Si le point de code est < 128, chaque octet est identique à la valeur du "
-"point de code."
+"Si le point de code est < 128, il est représenté par la valeur de l'octet "
+"correspondant."
 
 #: ../Doc/howto/unicode.rst:132
 msgid ""
 "If the code point is >= 128, it's turned into a sequence of two, three, or "
 "four bytes, where each byte of the sequence is between 128 and 255."
 msgstr ""
-"Si le point de code est ≥ 128, il est transformé en une séquence de deux, "
+"Si le point de code est >= 128, il est transformé en une séquence de deux, "
```

Comment:
Tu n'aime pas unicode ? `U+2265 GREATER-THAN OR EQUAL TO` me paraissait bien ici :p

Reply:
oui, je préfère comme ça :)
Je préfère éviter les symboles compliqués que les gens auront du mal à comprendre/refaire (comme l'espace insécable, je lui préfère ` : ::`)